pkg/apis/core/validation/validation.go

```diff
@@ -4229,15 +4229,13 @@ func ValidatePodEphemeralContainersUpdate(newPod, oldPod *core.Pod, opts PodVali
 	}
 	allErrs := validateEphemeralContainers(spec.EphemeralContainers, spec.Containers, spec.InitContainers, vols, specPath, opts)
 
-	// Existing EphemeralContainers may not be changed. Order isn't preserved by patch, so check each individually.
+	// Existing EphemeralContainers may not be changed, but can be removed. Order isn't preserved by patch, so check each individually.
 	newContainerIndex := make(map[string]*core.EphemeralContainer)
 	for i := range newPod.Spec.EphemeralContainers {
 		newContainerIndex[newPod.Spec.EphemeralContainers[i].Name] = &newPod.Spec.EphemeralContainers[i]
 	}
```

Comment:
Small nit: maybe `newEphemeralContainerIndex` instead of `newContainerIndex`?

Reply:
Updated. 
Thanks.